statsmodels/stats/tests/test_lilliefors.py

```diff
@@ -0,0 +1,26 @@
+from numpy.testing import assert_, assert_raises
+from scipy import stats
+import nose
+from statsmodels.stats._lilliefors import lilliefors
+import numpy as np
+
+class TestLilliefors(object):
```

Comment:
I haven't been able to find another Python package that implements it, but there are `R` packages.

There are more stringent tests with non-randomized data in `test_diagnostic.py`. I'll add tests for the exponential function to those as well.